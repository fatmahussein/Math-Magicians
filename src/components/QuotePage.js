import FetchQuote from './fetchQuote';

const QuotePage = () => (
  <FetchQuote />
);

export default QuotePage;
