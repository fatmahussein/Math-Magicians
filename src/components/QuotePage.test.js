import React from 'react';
import { render } from '@testing-library/react';
import QuotePage from './QuotePage';

test('QuotePage matches snapshot', () => {
  const { asFragment } = render(<QuotePage />);
  expect(asFragment()).toMatchSnapshot();
});
