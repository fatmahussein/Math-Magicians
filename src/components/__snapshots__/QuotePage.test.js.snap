// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuotePage matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="author"
    >
      <p>
        Loading...
      </p>
    </div>
  </div>
</DocumentFragment>
`;
